client/question/components/SpeechBalloonsContainerDirective.js

```diff
@@ -22,24 +22,17 @@ tie.directive('speechBalloonsContainer', [function() {
     scope: {},
     template: `
       <div>
-        <div class="tie-dot-container" ng-if="ConversationLogDataService.isNewBalloonPending()">
-          <div class="tie-dot tie-dot-1"></div>
-          <div class="tie-dot tie-dot-2"></div>
-          <div class="tie-dot tie-dot-3"></div>
-          <br>
-        </div>
-        <div ng-repeat="balloon in ConversationLogDataService.getSpeechBalloonsList() track by $index">
-          <div class="tie-speech-balloon-container">
+        <div ng-repeat="balloon in $parent.speechBalloonsList">
```

Comment:
Noted above, but just to confirm here: avoid $parent.

Reply:
Done